client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domains.tsx

```diff
@@ -46,7 +46,9 @@ const Domains: React.FC< Props > = ( { onSubmit } ) => {
 
 	const { __, _n } = useI18n();
 
-	const allGood = Object.values( domainsState ).every( ( { valid } ) => valid );
+	const allGood = Object.values( domainsState )
+		.filter( ( x ) => x.domain && x.auth )
```

Comment:
This would ignore errors and submit, is that on purpose?

Reply:
Well, the check for `.every( ( { valid } ) => valid );` is still there, so it shouldn't submit errors 🤔 
I think this will ignore rows without domain or auth code

Reply:
And this won't trigger the transfer if we have any not valid and not empty rows. I think this may be ok:
- if the user has some valid domains and some empty rows, then proceed
- if the user has some valid domains and some rows filled with data but invalid, do not proceed

WDYT @alshakero ?